packages/benchmarks/src/quizQuestions/QuizQuestionEval.ts

```diff
@@ -66,7 +66,6 @@ function makeQuizQuestionTask({
       stream: false,
       ...llmOptions,
       temperature: 0,
-      seed: 42, // Deterministically sample from the model, so same output every time
```

Comment:
Don't need the seed anymore? Will this affect outputs in a meaningful way?

Reply:
no, seed is meaningless when temperature is 0. only relevant when higher dep. cuz then makes the outputs deterministic for a given seed value. whereas for temp=0, output already deterministic